docs/Technical-Documentation/clean-and-reparse.md

```diff
@@ -0,0 +1,275 @@
+green  open   dev_ssp_m7_submissions_2024-07-05_17.26.26         CBNpxpgSS7agzFVg4YV8Wg   1   0          0            0       208b           208b
+green  open   dev_tanf_t1_submissions_2024-07-05_17.26.26        NRsIsXZbRMaZG7rERONtyA   1   0          2            0     17.6kb         17.6kb
+green  open   dev_tanf_t2_submissions_2024-07-05_17.26.26        tK8_BCgIR2iogOn4y9GLzA   1   0          2            0     16.9kb         16.9kb
+green  open   dev_tanf_t3_submissions_2024-07-05_17.26.26        MEpnURPnTwqItuQXEtc6sg   1   0          4            0     21.4kb         21.4kb
+green  open   dev_tanf_t4_submissions_2024-07-05_17.26.26        JLUCat_MQOG73lJyJ8SLcg   1   0          0            0       208b           208b
+green  open   dev_tanf_t5_submissions_2024-07-05_17.26.26        RI69-6F7QBKHMJ1LpXuyxA   1   0          0            0       208b           208b
+green  open   dev_tanf_t6_submissions_2024-07-05_17.26.26        r5FoR6QyTe-UXLC6tjvbbA   1   0          0            0       208b           208b
+green  open   dev_tanf_t7_submissions_2024-07-05_17.26.26        O8TxCKVHRJ66R7CD8k06OQ   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t1_submissions_2024-07-05_17.26.26 t_PQyvFfTLCLYWKDVQfcYQ   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t2_submissions_2024-07-05_17.26.26 CDRXMuEzRmuLJ7vutTq2dA   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t3_submissions_2024-07-05_17.26.26 RDVORzDnQs6t6q6Huhr2YQ   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t4_submissions_2024-07-05_17.26.26 oG-FtyMJSJKgAu-15476zg   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t5_submissions_2024-07-05_17.26.26 ABt7eBV_SbSYFm0wFxNoBA   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t6_submissions_2024-07-05_17.26.26 pKIIW7h8SXGwFwwJoiEwNA   1   0          0            0       208b           208b
+green  open   dev_tribal_tanf_t7_submissions_2024-07-05_17.26.26 crc6cuIHQfmxIUtY38Jrzw   1   0          0            0       208b           208b
+```
+
+- While the DAC reports the correct number of records for all submitted types, Elastic does not. This is because we only reparsed a subset of the entire collection of datafiles for the first time we executed the `clean_and_reparse` command. Therefore, Elastic only has documents for the subset of resubmitted files. If we had already executed the command: `python manage.py clean_and_reparse -a -n` and then executed `python manage.py clean_and_reparse -y 2024 -n`, we would see what you might have initially expected to see.
+
+```
+health status index                                              uuid                   pri rep docs.count docs.deleted store.size pri.store.size
```

Comment:
For clarity/conciseness, perhaps we can remove the columns that aren't docs.count and any records with a count of zero for these comparison sections?